content/pages/directory.md

```diff
@@ -4,9 +4,9 @@ Summary: Information about the SpaceAPI directory
 SortOrder: 40
 
 The directory hosts a list of hackerspaces which implement the SpaceAPI. It is
-currently maintained by the [FIXME Hackerspace](https://fixme.ch/).
+currently hosted by the [FIXME Hackerspace](https://fixme.ch/).
 
  * [Spaceapi directory](https://spaceapi.fixme.ch/directory.json)
  * Add your space to the directory by creating a pull request on
-   [Github](https://github.com/fixme-lausanne/OpenSpaceDirectory)
+   [Github](https://github.com/SpaceApi/OpenSpaceDirectory)
```

Comment:
Will we rename the repository (from OpenSpaceDirectory)? If yes what would be a good new name? @dbrgn ?

Reply:
It's here: https://github.com/SpaceApi/directory/

Reply:
Done.